domain-get-subdomains/index.ts

```diff
@@ -27,6 +28,10 @@ const httpTrigger: AzureFunction = async function (
       body: response,
     };
   } catch (err) {
+    context.log.error(
+      `GET Subdomains, error encountered: `,
```

Comment:
tiny itty bitty nit is that we don't actually do any interpolation here and can just use quotes instead of backticks

Reply:
Corrected backticks by replacing them with quotes